packages/vuetify/src/mixins/menuable/index.ts

```diff
@@ -165,9 +243,26 @@ export default baseMixins.extend<options>().extend({
 
   beforeMount () {
     this.hasWindow = typeof window !== 'undefined'
+
+    if (this.hasWindow) {
+      window.addEventListener('resize', this.onResize, false)
```

Comment:
Do we need to listen for the scroll event as well?

Reply:
I don't think it's necessary, the calculations don't use the scroll, everything is just offset relatively to the v-app now

I'm usually reluctant to add any scroll listeners, It would be a performance drawback that wouldn't see much benefit.
Here the calculation only uses the scroll to check for overflow to avoid positioning it somewhere that's outside of the view pane. It doesn't make much sense to recalculate that after, because either the user cannot see the full menu and scrolls to see the rest (so the position should not change) or he just scrolls away and doesn't want to see the content of the menu (so why would you try to keep it in the view when he scrolls?)

Keeping in mind that the menu is absolutely positioned and not fixed

Reply:
Ok.  I wasn't sure.  

I implemented a similar fix within my app: calculate top and side based on `getBoundingClientRect` of the v-app, then pass differences to the v-select's nudge properties.  When the view loads, the select is in the proper position, however it remains fixed in the same location after scrolling.  Perhaps I need to absolutely position it to solve the issue.

Reply:
It seems you are right, Element.getBoundingClientRect() method returns a DOMRect object providing information about the size of an element and its position **relative to the viewport**.

So the calculation would be wrong if you recalculate it with a scrolled window, I need to do some deeper testings

Reply:
Indeed

# Before with bounding rect
![image](https://user-images.githubusercontent.com/6115458/118862788-6748b200-b8de-11eb-87b0-27adb90513ae.png)

# With cumulative offset
![image](https://user-images.githubusercontent.com/6115458/118862866-7fb8cc80-b8de-11eb-937a-4a198b4d8911.png)